tests/Tests/EventStoreTest.php

```diff
@@ -364,7 +372,7 @@ public function it_can_process_the_all_stream_with_a_forward_iterator()
             'handler' => new \GuzzleHttp\Handler\CurlMultiHandler(),
         ]);
         $httpClient = new GuzzleHttpClient($client);
-        $this->es = new EventStore('http://127.0.0.1:2113', $httpClient);
+        $this->es = new EventStore('http://admin:changeit@127.0.0.1:2113', $httpClient);
```

Comment:
credentials can be passed to the underlying Guzzle client and look much better to me